vnext/src/Libraries/Text/Text.windows.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import DeprecatedTextPropTypes from '../DeprecatedPropTypes/DeprecatedTextPropTypes';
+import StyleSheet from '../../StyleSheet/StyleSheet';
+import Text from '../../Text/Text.js';
+import View from '../../Components/View/View';
+
+// [Windows]
+// Forking Text as a workaround in order to be able to render borders around it.
+function BorderedText(props) {
+  let { margin, padding, ...rest } = props.style != undefined
+    ? (Array.isArray(props.style)
+      ? StyleSheet.flatten(props.style)
+      : props.style)
+    : {};
+
+  let { style, ...textPropsLessStyle } = props;
+
+  if (props.style && props.style.borderWidth &&
+    props.style.borderColor) {
+    return (
+      <>
+        <View style={props.style}>
+          <Text style={rest} {...textPropsLessStyle} />
+        </View>
+      </>
+    );
+  } else {
+    return (
+      <>
+        <Text {...props} />
+      </>);
```

Comment:
not a jsx expert here, so question: my understanding is that the `<>` syntax allows returning a collection of children instead of just one. However in both cases you have just one item (with one or zero children respectively). So is wrapping them in a fragment necessary?

Reply:
You're right, it's not necessary, I'll remove it